runtime/src/main/java/org/corfudb/runtime/object/transactions/AbstractTransactionalContext.java

```diff
@@ -79,6 +79,7 @@
     public final TransactionBuilder builder;
 
     /**
+     * TODO remove this!
```

Comment:
The completion time is currently used for a log.trace() message, I think.  If making it more official'y via a new metering statistic is useful, I'm fine with that.  I'll let y'all armwrestle about wisdom of other uses.